### ISSUE TITLE
Installation failure

### ISSUE THREAD

**Issue by verdurin** [kaneplusplus/synchronicity#4]
I'm trying to build synchronicity using EasyBuild and it's failing with this error:

```
g++ -I/opt/soft/apps/R/3.2.3-foss-2015b/lib64/R/include -DNDEBUG -I../inst -DLINUX -I/opt/soft/apps/OpenBLAS/0.2.14-GNU-4.9.3-2.25-LAPACK-3.5.0/include -I/opt/soft/apps/ScaLAPACK/2.0.2-gompi-2015b-OpenBLAS-0.2.14-LAPACK-3.5.0/include -I/opt/soft/apps/FFTW/3.3.4-gompi-2015b/include -I/opt/soft/apps/libreadline/6.3-foss-2015b/include -I/opt/soft/apps/ncurses/5.9-foss-2015b/include -I/opt/soft/apps/libpng/1.6.17-foss-2015b/include -I/opt/soft/apps/libjpeg-turbo/1.4.1-foss-2015b/include -I/opt/soft/apps/Java/1.8.0_72/include -I/opt/soft/apps/Tcl/8.6.4-foss-2015b/include -I/opt/soft/apps/Tk/8.6.4-foss-2015b-no-X11/include -I/opt/soft/apps/cURL/7.45.0-foss-2015b/include -I/opt/soft/apps/libxml2/2.9.2-foss-2015b/include -I/opt/soft/apps/GDAL/2.0.1-foss-2015b/include -I/opt/soft/apps/PROJ/4.8.0-foss-2015b/include -I"/opt/soft/apps/BH/1.60.0-1-foss-2015b-R-3.2.3/BH/include" -I"/opt/soft/apps/R/3.2.3-foss-2015b/lib64/R/library/Rcpp/include"   -fpic  -O2 -march=native  -c synchronicity.cpp -o synchronicity.o
synchronicity.cpp: In function ‘boost::posix_time::ptime to_ptime(long int)’:
synchronicity.cpp:144:10: error: ‘second_clock’ has not been declared
   return second_clock::local_time() + seconds( timeout );
          ^
make: *** [synchronicity.o] Error 1
ERROR: compilation failed for package ‘synchronicity’
```

In case this was something to do with EasyBuild, I looked at the package on CRAN. It reports on the package page that it's passing test, but on looking more closely you can see it tails the CRAN checks:

https://cran.r-project.org/web/checks/check_results_synchronicity.html

and you see the same error at:

https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-gcc/synchronicity-00install.html

```
** libs
make[1]: Entering directory '/data/gannet/ripley/R/packages/tests-devel/synchronicity/src'
g++ -I/data/gannet/ripley/R/R-devel/include -DNDEBUG -I../inst -DLINUX -I/usr/local/include -I"/data/gannet/ripley/R/test-3.3/BH/include" -I"/data/gannet/ripley/R/test-3.3/Rcpp/include"   -fpic  -g -O2 -Wall -pedantic -mtune=native  -c RcppExports.cpp -o RcppExports.o
g++ -I/data/gannet/ripley/R/R-devel/include -DNDEBUG -I../inst -DLINUX -I/usr/local/include -I"/data/gannet/ripley/R/test-3.3/BH/include" -I"/data/gannet/ripley/R/test-3.3/Rcpp/include"   -fpic  -g -O2 -Wall -pedantic -mtune=native  -c SharedCounter.cpp -o SharedCounter.o
g++ -I/data/gannet/ripley/R/R-devel/include -DNDEBUG -I../inst -DLINUX -I/usr/local/include -I"/data/gannet/ripley/R/test-3.3/BH/include" -I"/data/gannet/ripley/R/test-3.3/Rcpp/include"   -fpic  -g -O2 -Wall -pedantic -mtune=native  -c synchronicity.cpp -o synchronicity.o
g++ -I/data/gannet/ripley/R/R-devel/include -DNDEBUG -I../inst -DLINUX -I/usr/local/include -I"/data/gannet/ripley/R/test-3.3/BH/include" -I"/data/gannet/ripley/R/test-3.3/Rcpp/include"   -fpic  -g -O2 -Wall -pedantic -mtune=native  -c util.cpp -o util.o
synchronicity.cpp: In function ‘boost::posix_time::ptime to_ptime(long int)’:
synchronicity.cpp:144:10: error: ‘second_clock’ has not been declared
   return second_clock::local_time() + seconds( timeout );
          ^
/data/gannet/ripley/R/R-devel/etc/Makeconf:141: recipe for target 'synchronicity.o' failed
make[1]: *** [synchronicity.o] Error 1
make[1]: Target 'all' not remade because of errors.
make[1]: Leaving directory '/data/gannet/ripley/R/packages/tests-devel/synchronicity/src'
ERROR: compilation failed for package ‘synchronicity’
* removing ‘/data/gannet/ripley/R/packages/tests-devel/synchronicity.Rcheck/synchronicity’
```

**Comment by kaneplusplus** [comment]
Hi @verdurin thanks for letting me know. I'll take a look and report back in the next couple of days.

**Comment by madeleineth** [comment]
This is failing for me, too. It looks like it's fixed in 5e5b790a96d77684df15090330bcacd82c2bd8f4, but that's not on CRAN.

**Comment by kaneplusplus** [comment]
The new version is up on CRAN. Thanks for the reminder.
